modules/light-clients/08-wasm/testing/values.go

```diff
@@ -26,9 +32,24 @@ var (
 	ErrMockVM       = errors.New("mock vm error")
 )
 
+// CreateMockWrappedClientState returns valid wrapped client state for use in tests.
+func CreateMockWrappedClientState(height clienttypes.Height) *ibctm.ClientState {
```

Comment:
```suggestion
// CreateMockTmClientState returns valid wrapped client state for use in tests.
func CreateMockTmClientState(height clienttypes.Height) *ibctm.ClientState {
```

I understand the intention here. But the naming of these functions are confusing imo.

Reply:
Renamed!